sphinx_js/doclets.py

```diff
@@ -45,8 +48,16 @@ def gather_doclets(app):
                 d)
         except PathTaken as conflict:
             conflicts.append(conflict.segments)
+        except parsimonious.exceptions.ParseError as e:
+            if not app.config.sphinx_js_lax:
+                raise
+            else:
+                print('Could not parse correctly %s' % d)
```

Comment:
This PR seems reasonable, but let's use Sphinx's logging API rather than print(): https://www.sphinx-doc.org/en/master/extdev/logging.html